### PR TITLE
Rob branch

### PR DESCRIPTION
**Pull request by rob166**
Added Login and Signup to App.js

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -3,28 +3,61 @@ import Navbar from './components/Navbar.js'
 import Product from './routes/Product.js';
 import Landing from './routes/Landing.js';
 import Home from './routes/Home.js'
+import Signup from './routes/Signup'
+import Login from './routes/Login.js'
+import { useState } from 'react';
+
 
 function App() {
+  const BASE_URL = 'https://localhost:5432/grace-shopper/api';
```
Comment (DannyReese) [inline client/src/App.js:12]: the base url is http://localhost:3001/api

**Review by DannyReese — CHANGES_REQUESTED** [review]
looks good just the base url needs to change

File: client/src/App.js (hunk 1 above)
Comment (rob166) [reply client/src/App.js]: Just updated, good catch thanks

**Comment by rob166** [comment]
Updated BASE_URL

**Review by DannyReese — APPROVED** [review]
Looks good

**Pull request merged**